### DISCUSSION TITLE
client->upload throws a 'BadRequest' exception

### DISCUSSION THREAD

**Discussion by rabol** [spatie/ray#468] [Q&A]
Hi

I'm trying to upload an file to dropbox
I have created an access token in Dropbox
and use the following code:

```
$client = new Client($authorizationToken);

$file = 'pdf/' . 'testfile.pdf';
if (Storage::exists($file)) {
    $res = $client->upload('/', Storage::get($file));
} 
```
it throws a 'Bad Request' exception

Spatie\Dropbox\Client::determineException
vendor/spatie/dropbox-api/src/Client.php:665

Not really sure why

**Comment by freekmurze** [comment]
Seems unrelated to this repo.

**Reply by rabol** [reply]
yes, I saw that the repo changed-
I started the issue from
https://github.com/spatie/flysystem-dropbox/issues/new/choose

and pressed 'ask a question'

**Reply by rabol** [reply]
have a look at the screen shot:

the url in the top

and at the bottom the url for the 'open'

![Screenshot 2021-06-02 at 14 24 59](https://user-images.githubusercontent.com/1177191/120479962-d3cbb280-c3ae-11eb-9d50-fe00893d50bd.png)
